### ISSUE TITLE
System.Text.Encoding GetBytes returns a Uint8Array

### ISSUE THREAD

**Issue by sichbo** [bridgedotnet/Bridge#3127]
This one's a bit of a gotcha :) 

`Encoding.GetBytes` returning a Uint8Array isn't necessarily a bad thing when it fits particular use cases, but the expectation from C# is that what you're holding onto is a byte[]. When you try to use array functions such as `Array.Concat` the outcome is unexpected:

https://deck.net/e6d00a3199ebfc46866090e64b5d1212

```c#
public class Program
{
    public static void Main()
    {
        // A workaround (at cost of performance) is to .ToArray<byte>();
        // on bytes.
        var bytes = System.Text.Encoding.UTF8.GetBytes("12345678");
        var ar = new byte[]{1,2,3};
        ar = (byte[])ar.Concat(bytes);
        Console.WriteLine(ar);
    }
}
```
Yields:

```
1,2,3,[object Uint8Array]
```
Should be:
```
1,2,3,49,50,51,52,53,54,55,56
```

**Comment by LeonidVeriga** [comment]
@sichbo, yes, the decision to have typed array as an output of **GetBytes()** may be in question. However, your sample works for me as expected (I tried it on Chrome and FF).
Are there any specific steps to reproduce?

**Comment by LeonidVeriga** [comment]
Ok, looks like good old **IE**.

**Comment by geoffreymcgill** [comment]
Our plan is to fix this defect and include in the 16.3.1 patch.

@sichbo – Thanks for the report.

**Comment by sichbo** [comment]
Thanks for the update.
